.github/ISSUE_TEMPLATE/resourcescheduler.yml

```diff
@@ -0,0 +1,46 @@
+name: Request to schedule events to increase compute
+description: Schedule specific dates/times when your coursework requires higher computing resources
+labels: "support"
+title: Increase compute resources for Course XXX between specific date(s)/time(s)
+assignees:
+  - shaneknapp
+  - sean-morris
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please follow the sections in the template and provide all the required information for us to validate and allocate increased compute for your class.
+  - type: textarea
+    attributes:
+      label: Course Name
```

Comment:
Maybe this should just be the hub they are using? I don't if we care about the course name